cpp/data/frameProcessor/src/XspressProcessPlugin.cpp

```diff
@@ -345,6 +347,26 @@ void XspressProcessPlugin::setup_memory_allocation()
     free(inp_est_memblock_);
   }
   inp_est_memblock_ = malloc(sizeof(double) * this->frames_per_block_ * num_channels_);
+
+  if (sum_memblock_)
+  {
+    free(sum_memblock_);
+  }
+  sum_memblock_ = malloc(sizeof(double) * this->frames_per_block_);
+  }
```

Comment:
This bracket being indented really confused me

Reply:
My bad, I did the indent in a block and got the bracket by accident ):